[DOCS] Proofreading developer documentation moved from wiki. (#15886)
openvinotoolkit/openvino: src/common/snippets/docs/snippets_cpu_target.md

# CPU Target for SnippetS Code Generator

> Snippets in its first generation can be seen as a generalization over a generic eltwise node. First generation of snippets has a lack of integration with oneDNN, and the patterns it supports should be kept orthogonal to what is fused with post-ops.

Author: ```suggestion
Snippets in its first generation can be seen as a generalization over a generic eltwise node. First generation of snippets does not have integration with oneDNN, and the patterns it supports should be kept orthogonal to what is fused with post-ops.
```
